scratch/camera_feed/display_camera_feed.cpp

```diff
@@ -0,0 +1,235 @@
+#include "../../src/network/network.hpp"
+#include <stdlib.h>
+#include <string.h>
+#include <iostream>
+#include <turbojpeg.h>
+#include "SDL2/SDL.h"
+//#include <inttypes.h> to print uints
+// If you are trying to print a uintxx_t there are 2 methods I have found.
+/*
+    std::cout << unsigned(uint you are trying to print) << std::endl;
+    printf("%" PRIu16 "\n", uint you are trying to print);
+ */
+struct BufferItem{
```

Comment:
if you change it to `typedef Struct Foo_s {}; Foo` you can type `Foo` instead of `struct Foo` around your code base